### ISSUE TITLE
Updating from 1.0.25 to 1.0.27 lost all my notes on Windows!

### ISSUE THREAD

**Issue by SweRavn** [laurent22/joplin#1223]
### Operating system

- Windows

### Application

- Desktop

**Comment by laurent22** [comment]
So there's no question?

**Comment by SweRavn** [comment]
Well, the title says it all - after updating all notes were lost. That is the issue.

**Comment by laurent22** [comment]
If you provide more info it might be possible to help - what's on the sync target? Are the notes still there? How about the profile directory on your computer, is there any data in there? I doubt something as extreme as deleting all the data could happen during an update, so more info is needed basically.

**Comment by SweRavn** [comment]
You are probably right. I am not sure what happened. When staring Joplin now, there were notes with welcome messages and example of what can be done. But all my old notes that I had are not shown in the application. I have not set any synchronization, only running on local machine. I have no clue where the files corresponding to the application resides, so I cannot check if they are there or not.

I installed 1.0.27 for all users, not sure how it was before. That is the only thing I can think of that might be related to the issue.

**Comment by laurent22** [comment]
My guess is that your previous app was using one profile in one location, and the new install using a profile in a different location (thus a blank one with empty notes). You'd need to search for "joplin-desktop" on your computer to find where your previous profile and the new one is.

**Comment by SweRavn** [comment]
Sorry, I'm not saying this to be rude, but as a user, it is not okay that when I do an update, the notes stop appearing in the app. I do not want to have to search around the file system from some directory which I have no clue what to do with. I managed to find a joplin-desktop directory in my home directory in the .config folder. But I have no clue what to do with that. I opened the sqlite db in a generic sqlite db-viewer and my notes are there, but I still think this is a bug in the software, it should not stop displaying those notes, and how to make the application now show them?
